src/components/Strategies/Blocks/index.ts

```diff
@@ -10,6 +11,7 @@ const blockConfig: {
 } = {
   'arbitrum:bridgeworld:claim': { component: Arbitrum_Bridgeworld_Claim, requiresFallback: false },
   'arbitrum:magic:send': { component: Arbitrum_Magic_Send, requiresFallback: true },
+  'arbitrum:magicdragon:claim': { component: Arbitrum_MagicDragon_Claim, requiresFallback: true },
```

Comment:
`requiresFallback` should be `false`. it should only be `true` for transactions that might not be possible to execute because you don't have enough balance in your wallet. claiming can always be done